nix/download.tt

```diff
@@ -4,31 +4,36 @@
 
 <p>The latest and recommended version of Nix is
 <strong>[%latestNixVersion%]</strong>. The quickest way to install it
-on Linux and Mac OS X is to run the following in a shell (as a user
-other than <tt>root</tt>):
+on Linux and macOS is to run the following in a shell (as a user
+other than <tt>root</tt>):</p>
 
-<pre><span class="muted">$</span> curl <a href="[%root%]nix/install">https://nixos.org/nix/install</a> | sh</pre>
+<pre>
+<span class="command">curl https://nixos.org/nix/install | sh</span>
+</pre>
 
-Make sure to follow the instructions output by the script.
+<p>Make sure to follow the instructions output by the script.</p>
 
-You may want to verify the integrity of the installation script using GPG:
+<h3>GPG Verification</h3>
+<p>You may want to verify the integrity of the installation script
+using GPG:</p>
 
 <pre>
-<span class="muted">$</span> curl -O <a href="[%root%]nix/install">https://nixos.org/nix/install</a>
-<span class="muted">$</span> curl -O <a href="[%root%]nix/install.sig">https://nixos.org/nix/install.sig</a>
-<span class="muted">$</span> gpg2 --verify ./install.sig
-<span class="muted">$</span> sh ./install
+<span class="command">curl -o install-nix-[%latestNixVersion%] https://nixos.org/nix/install</span>
+<span class="command">curl -o install-nix-[%latestNixVersion%].sig https://nixos.org/nix/install.sig</span>
+<span class="command">gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE</span>
+<span class="command">gpg2 --verify ./install-nix-[%latestNixVersion%].sig</span>
+<span class="command">sh ./install-nix-[%latestNixVersion%]</span>
 </pre>
 
-The <a href="[%root%]edolstra.gpg">signing key</a> has fingerprint <tt>B541 D553 0127 0E0B CF15  CA5D 8170 B472 6D71 98DE</tt>.</p>
+<p>The <a href="[%root%]edolstra.gpg">signing key</a> has fingerprint <tt>B541 D553 0127 0E0B CF15  CA5D 8170 B472 6D71 98DE</tt>.
+It is also available <a href="https://github.com/NixOS/nixos-homepage/blob/master/edolstra.gpg">on GitHub</a>.</p>
 
 <p>The installation script requires that you have <tt>sudo</tt> access
-to <tt>root</tt>, unless the directory <tt>/nix</tt> already exists
-and is writable by you. You can uninstall Nix simply by running <tt>rm
+to <tt>root</tt>. You can uninstall Nix simply by running <tt>rm
```

Comment:
The part about `/nix` existing / owned by you isn't true when on macOS, so I just deleted it for simplicity.